spec/lumberjack_spec.rb

```diff
@@ -97,7 +99,7 @@
   context "when compiled from source" do
     let(:lsf) do
       # Start the process, return the pid
-      IO.popen(["./logstash-forwarder", "-config", config_file, "-quiet"])
+      IO.popen(["./logstash-forwarder", "-config", config_file, "-quiet", "-from-beginning=true"])
```

Comment:
`-from-beginning` isn't a flag anymore. Valid flags are available in `./logstash-forwarder -h` or visible in the code, here: https://github.com/elastic/logstash-forwarder/blob/master/logstash-forwarder.go#L64-L81